desc/basis.py

```diff
@@ -1726,3 +1726,180 @@ def zernike_norm(l, m):
 
     """
     return np.sqrt((2 * (l + 1)) / (np.pi * (1 + int(m == 0))))
+
+
+def FourierZernike_to_PoincareZernikePolynomial(X_lmn_3D, basis_3D, zeta=0):
```

Comment:
would these be better as methods of the relevant class?

like `FourierZernikeBasis.to_ZernikePolynomial`